question1/README.md

```diff
@@ -9,5 +9,13 @@ where npts contain the number of values in array x, and max_ptr and min_ptr are
 Provide and print out 3 examples to show that your function is working as intended.
 
 Compile Steps: 
-
+gcc q11.c
 Output:
+How many numbers do you wanna input ?5
+Enter 5 numbers to find the maximum and minimum1
+2
+3
+4
+5
```

Comment:
you should have 2 other examples; -2
18/20

no need to commit the compiled files